csrc/ir/internal_base_nodes.h

```diff
@@ -456,6 +456,8 @@ class TORCH_CUDA_CU_API TensorDomain : public Val {
 
   TensorDomain(const TensorDomain* src, IrCloner* ir_cloner);
 
+  static TensorDomain* duplicateTensorDomain(const TensorDomain* src);
```

Comment:
Does it make sense to make it a member function `clone()`?

Reply:
I'll go with `duplicate` instead of `clone`. simply because we have IrCloner, which associates with copying things across containers...

Terminology is really confusing for us mortals. :kneeling_person: